### PR TITLE
Run Python 3.5 tests with 3.5.2

### PR DESCRIPTION
**Pull request by dilipped**

### CODE REVIEW

File: test/ci/kokoro/run_integ_tests.sh [hunk 1]
```diff
@@ -42,6 +42,14 @@ if [[ $KOKORO_JOB_NAME =~ "macos" ]]; then
 fi
 
 function latest_python_release {
```
Comment (thomasmaclean) [inline test/ci/kokoro/run_integ_tests.sh:44]: Can we change this to "preferred_python_release" to reduce later confusion? There are comments further down clarifying that they're getting the "latest" available, so I think everything will still semantically make sense after the change.

**Review by thomasmaclean — CHANGES_REQUESTED** [review]

File: test/ci/kokoro/run_integ_tests.sh (hunk 1 above)
Comment (dilipped) [reply test/ci/kokoro/run_integ_tests.sh]: Good point! Done.

Comment (dilipped) [reply test/ci/kokoro/run_integ_tests.sh]: Oops, missed updating the calling function

Comment (dilipped) [reply test/ci/kokoro/run_integ_tests.sh]: Now it's done.

**Review by thomasmaclean — APPROVED** [review]

**Pull request merged**